app/views/components/gds/gds_main_template.scala.html

```diff
@@ -30,7 +30,8 @@
 
 @(  title: Title,
     backButtonTitle: String = "Back",
-    backButtonUrl: String = "/")(contentBlock: Html)(implicit request: Request[_], messages: Messages)
+    backButtonUrl: String = "/",
+    requiresBackLink: Boolean = true)(contentBlock: Html)(implicit request: Request[_], messages: Messages)
```

Comment:
You have two fields here when you could just have an `Option[String]`

Reply:
I could, but I didn't want to break existing views / force them to define Some(url).  Having a back button is the norm.  I think it's only the confirmation pages that don't need one.

Reply:
Intergrated with Option[BackButton] solution